mod/allfriends.php

```diff
@@ -8,13 +8,12 @@
 use Friendica\Core\L10n;
 use Friendica\Core\System;
 use Friendica\Database\DBA;
-use Friendica\Model\Contact;
-use Friendica\Model\GContact;
-use Friendica\Model\Profile;
+use Friendica\Model;\
+use Friendica\Module\Contact;
```

Comment:
Although this is correct, please write `use Friendica\Module;` and later `Module\Contact::getTabsHTML` to prevent any immediate confusion.